src/Components/PageNotFound.jsx

```diff
@@ -0,0 +1,15 @@
+import Button from "react-bootstrap/Button";
+import { useNavigate } from "react-router-dom";
+
+const PageNotFound = () => {
```

Comment:
This component is fine as it currently is, but it might be nice to pass the error into here, then sent a more tailored message 👍